actions/common-prerequisites/action.yml

```diff
@@ -16,9 +16,9 @@ runs:
         cache: gradle
 
     - name: Install Cocogitto
-      uses: cocogitto/cocogitto-action@c7a74f5406bab86da17da0f0e460a69f8219a68c # v3.11
+      uses: cocogitto/cocogitto-action@9a9fe03b31c47444290c0d7f9b1ee1b44ee13f20 # v4.1.0
```

Comment:
I needed to update the Cocogitto action to get Cocogitto [6.5.0](https://github.com/cocogitto/cocogitto/releases/tag/6.5.0) which gives us the new `--include-prereleases` flag.

> https://github.com/cocogitto/cocogitto/commit/55fb7f0dc0d2a8f44c2ca11db90bc1bc0275895b - (get-version) add options to print full tag and include prerelease versions